app-admin/kio-admin/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>kde@gentoo.org</email>
+		<name>Gentoo KDE Project</name>
+	</maintainer>
+	<upstream>
+		<bugs-to>https://bugs.kde.org/</bugs-to>
```

Comment:
```suggestion
		<bugs-to>https://bugs.kde.org/enter_bug.cgi?product=kio-admin</bugs-to>
```

Reply:
Do you intend that other packages in tree also follow this convention?

Reply:
Where possible and where we stumble over it, yes. @thesamesam?